app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    return phrase.lower().count(letter.lower())
```

Comment:
The function `count_occurrences` is currently not doing anything except returning the count of the letter in the phrase. The function body is empty. Please make sure to implement the function according to the task requirements.